fortran/test/fetch_content_integration/CMakeLists.txt

```diff
@@ -68,6 +68,22 @@ if (MUSICA_ENABLE_MICM)
     COMMAND $<TARGET_FILE:test_get_micm_version>
     WORKING_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
   )
+
+  target_link_libraries(test_micm_box_model
+    PRIVATE
+      musica::musica-fortran
+  )
```

Comment:
I think you might need to add this line:
```
add_exectuable(test_micm_box_model test_micm_box_model.F90)
```
unless I'm just missing where this is